roles/solr9cloud/tasks/backup_gcs_mount.yml

```diff
@@ -0,0 +1,173 @@
+---
+# Configure GCS backup mount for SolrCloud using gcsfuse
+- name: Solrcloud | Install gcsfuse dependencies
```

Comment:
Would the first few tasks here make sense in their own gcsfuse role?  Definitely not a blocker.

Reply:
they would. I think we should do it "when" (and if we remember) we have another use for gcsfuse